main.w

```diff
@@ -192,9 +193,10 @@ new cloud.OnDeploy(inflight () => {
 }) as "InitializeTable";
 
 let api = new cloud.Api(cors: true) as "VotingAppApi";
+let localStackApiURL = cdktf.Fn.replace(api.url, "us-east-1.amazonaws.com", "localhost.localstack.cloud:4566");
```

Comment:
`api.url` is https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_deployment#invoke_url and localstack doesn't seem to be configured there for some reason